src/builder.rs

```diff
@@ -0,0 +1,110 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+use hyper::{Body, Request, Response};
+
+use crate::app_context::AppContext;
+use crate::async_graphql_hyper::{GraphQLBatchRequest, GraphQLRequest};
+use crate::blueprint::{validate_operations, Blueprint, OperationQuery};
+use crate::config::{Config, ConfigModule, ConfigReader, Source};
+use crate::fmt::Fmt;
+use crate::http::handle_request;
+use crate::print_schema;
+use crate::runtime::TargetRuntime;
+use crate::valid::Validator;
+
+#[derive(Clone)]
+pub struct TailcallBuilder {
+    files: Vec<String>,
+    schemas: Vec<SchemaHolder>,
+    runtime: TargetRuntime,
+}
+#[derive(Clone)]
+struct SchemaHolder {
+    source: Source,
+    schema: String,
+    parent: Option<String>,
```

Comment:
I don't think we need `parent`.

Reply:
Add docs about each field. It wasn't clear the first time that `parent` is actually file path.